src/components/layout/nav/components/partner/nav-partner-desktop.tsx

```diff
@@ -23,6 +23,13 @@ type NavPartnerDesktopProps = {
     hide_login_signup: boolean
 }
 
+type StyledNavRightProps = {
+    is_rtl: boolean
+    mounted: boolean
+    move: boolean
+    // button_ref: any
```

Comment:
Do we need this `// button_ref: any`?

Reply:
Yes, we are using this here, but I'm not sure about it's type, and I don't want to add **any** type. so I commented this for now, will fix that later!